requirements.txt

```diff
@@ -1,23 +1,19 @@
-Django==1.11.29
+Django==3.2
```

Comment:
Let's set this to `3.2.13` specifically (or whatever the latest release is before we merge!)

Reply:
Done! builds and runs fine.